### PR TITLE
[ULMS-1911] Add docs

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

File: docs/src/workflow.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    opt auth timeout
+        Agent ->> Presence: ws connect
+        activate Agent
+        activate Presence
+        Presence ->> Presence: wait for 10s
+        Presence ->> Agent: close connection
+        deactivate Agent
+        deactivate Presence
+    end
```
Comment (manifest) [inline docs/src/workflow.md:20]: Authn/authz failure workflow.

File: docs/src/workflow.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+    Nats ->> Presence: new event
+    Presence ->> Agent: new event
+
+    opt pong timeout
+        Presence->>Agent: ping
+        Presence->>Presence: wait for 5s
+        Presence->>Agent: close connection
+    end
```
Comment (manifest) [inline docs/src/workflow.md:44]: Ping timeout workflow.

File: docs/src/agent/events.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Arrives when someone leaves the classroom
+
+Subject: `classrooms.{:CLASSROOM_ID}.presence`
+
+Attribute | Type   | Optional  | Description
+----------| ------ |-----------| -----------
+kind      | string |           | "event"
+label     | string |           | "agent.leave"
+payload   | string | +         | Agent ID
```
Comment (manifest) [inline docs/src/agent/events.md:25]: Error notifications are missing here. I guess we may place "workflow/sequence-diagram" related to each particular event here.

File: docs/src/workflow.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    actor Agent
+    participant Presence
+    participant Nats
+    participant DB
+
+    Agent ->> Presence: close connection
+    activate Presence
+    Presence ->> Nats: agent.leave
+    Presence ->> DB: move Agent session to history
+    Presence ->> Presence: close connection for Agent
```
Comment (manifest) [inline docs/src/workflow.md:64]: Hasn't it been already closed by the user agent?

File: docs/src/authz.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+URI of authorization endpoint, object and anonymous namespaces are configured through the application configuration file.
+
+Possible values for `OBJECT` and `ACTION`:
+
+Object                       | Action  | Description
+-----------------------------|---------| -----------
+["classrooms"]               | read    | A service counts online agents.
+["classrooms", CLASSROOM_ID] | read    | An user reads information about the number of online agents in the classroom.
+["classrooms", CLASSROOM_ID] | connect | An user connects to the classroom.
```
[the remark this answers is not in the document]
Comment (mgrachev) [reply docs/src/authz.md:31]: I don't know. Anyway, the dispatcher transforms it into [read](https://github.com/foxford/dispatcher/blob/master/src/app/api/v1/authz/mod.rs#L340).

**Review by manifest — APPROVED** [review]

**Pull request merged**